src/components/FiledRequestCard/FiledRequestCard.tsx

```diff
@@ -0,0 +1,81 @@
+import { Box, Button, Card } from '@mui/material'
+import styles from './style'
+import {FiledRequest} from "@interfaces/filedRequest";
+
+// Just copied over from InsightCard as first draft, field layout will be adjusted
+interface FiledRequestCard {
+    filedRequestCardData: FiledRequest,
+    onDelete: Function
```

Comment:
Please add a return type for the function, maybe void if we're not expecting anything

Reply:
Done